db/migrate/20230606140456_create_like.rb

```diff
@@ -2,7 +2,7 @@ class CreateLike < ActiveRecord::Migration[7.0]
   def change
     create_table :likes do |t|
       t.string :timestamp
-      t.references :users, null: false, foreign_key: true, index: true
+      t.references :authors, null: false, foreign_key: true, index: true
```

Comment:
- Please change reference to: `t.references :author, null: false, foreign_key: { to_table: 'users' }, index: true`. Do the same for the other migration files. For `Post` and `Comment`.